cinema/views.py

```diff
@@ -46,7 +47,7 @@ class ActorViewSet(
 ):
     queryset = Actor.objects.all()
     serializer_class = ActorSerializer
-    authentication_classes = (TokenAuthentication,)
+    # authentication_classes = (TokenAuthentication,)
```

Comment:
The `authentication_classes` is commented out in `ActorViewSet`. If authentication is required for these endpoints, consider uncommenting this line.